cabal.project

```diff
@@ -17,6 +17,13 @@ index-state:
 
 packages: **/*.cabal
 
+source-repository-package
+  type: git
+  location: https://github.com/cardano-scaling/haskell-accumulator
+  tag: 66db72a036fe894f4b8108fe64279d1bcc0e0b25
```

Comment:
Using specific commit, as I have some fixes needed in the repo: https://github.com/cardano-scaling/haskell-accumulator/pull/4